index.html

```diff
@@ -1,36 +1,37 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8">
-    <title>Startrly</title>
-  </head>
-  <body>
-    <div>
-
+<head>
+  <link href="normalize.css" rel="stylesheet"/>
+  <link href="startly_style.css" rel="stylesheet"/>
+  <meta charset="utf-8">
+  <title>Startrly</title>
+</head>
+<body>
+  <div class="container">
 
       <nav>
-        <ul>
+        <ul class="topper">
           <li>
             Startrly
           </li>
-          <li>
+          <li id="try">
             Try Now!
           </li>
-          <li>
+          <li id="about">
             About
           </li>
         </ul>
       </nav>
 
 
-      <section>
+      <section class="banner">
         <h1>Startrly</h1>
         <sub>A catchy tagline for a great solution</sub>
         <a href="#">Some Call to Action</a>
       </section>
 
 
-      <section>
+      <section class="quote">
```

Comment:
I like that you added in classes and id's, and reused them (like this one) to format the other quote too.